src/gitServices/github/GitHubService.ts

```diff
@@ -19,6 +21,23 @@ export class GitHubService implements GitService {
     }
 
     public async injectButton() {
+        await this._injectButton();
+
+        // GitHub uses Turbo to load the project repo's `Code`, `Issues`, `Pull requests`,
+        // `Actions`, etc. pages. In case the user clicks from a non-Code page to the Code
+        // page, try to inject button.
+        document.addEventListener("turbo:load", () => {
```

Comment:
If GitHub stops using https://github.com/hotwired/turbo in the future, nothing should break, but we would probably still experience https://github.com/redhat-developer/try-in-web-ide-browser-extension/issues/13